### DISCUSSION TITLE
connectPerpheral error connection error

### DISCUSSION THREAD

**Discussion by 8bottle** [innoveit/react-native-ble-manager#1024] [General]
Hi, 

Thanks for the great library work. We're getting an error when trying to connect on the device. 

Kindly check image attached. 

Thanks! 
![WhatsApp Image 2023-06-08 at 3 49 55 PM](https://github.com/innoveit/react-native-ble-manager/assets/65874404/3f2104b7-e180-4f6f-a5f3-31528cc2ca96)

**Comment by marcosinigaglia** [comment]
Check the permission and the device log.
